coalib/results/Diff.py

```diff
@@ -100,6 +100,20 @@ def __len__(self):
         return len(self._changes)
 
     @property
+    def changes_size(self):
+        """
+        Returns total number of changed lines, including lines that will be
+        added as changed.
+        """
+        size = 0
+        for line_diff in self._changes.values():
+            if line_diff.change or line_diff.delete:
+                size += 1
+            if line_diff.add_after:
+                size += len(line_diff.add_after)
+        return size
```

Comment:
The `__len__` may be used already in many places with the current intention and tests may not cover it, so changing it can be risky. And to rewrite with sum it'll be
```python
sum(len(line_diff.add_after) for line_diff in self._changes.values()
        if line_diff.add_after else 1) + \
len(line_diff for line_diff in self._changes.values()
        if line_diff.change or line_diff.delete)
```
I'm not sure this is simpler to read.